assets/css/campfire-games.css

```diff
@@ -0,0 +1,42 @@
+h2 {
+  margin-bottom: 20px;
+}
+
+.container {
+  padding-top: 30px;
+}
+
+#teams-container {
+  background-color: rgb(227,219,203);
+}
+
+.marshie {
+  height: 200px;
+  width: 200px;
```

Comment:
```suggestion
```
The green marshie on my screen looks WiDe because the PNG image isn't square. Rather than create a new version of the image that's square, we can actually just remove the `width` property. When we do that, `width` will default to `auto`, which will preserve the original aspect ratio of the image and make the green marshie normal :).
![image](https://user-images.githubusercontent.com/6632788/109360688-a1616500-7855-11eb-8968-dd548c43e318.png)